### DISCUSSION TITLE
About a precise citation formatting.

### DISCUSSION THREAD

**Discussion by thomasmansencal** [colour-science/colour#693] [Ideas]
-   APA
-   ISO 960
-   MLA

**Comment by thomasmansencal** [comment]
Admitting we use _APA_ formatting:

```
Smits, B. (1999). An RGB-to-spectrum conversion for reflectances. Journal of Graphics Tools, 4(4), 11-22.
```

In the code right now, I'm formatting the citation as follows:

```
.. [1]  **Smits, B. (1999)**,
        *An RGB-to-spectrum conversion for reflectances*,
        Journal of Graphics Tools, 4(4), 11-22.
        DOI: http://dx.doi.org/10.1080/10867651.1999.10487511
```

The big changes are:
- Presence of `**` and `*` for emphasis of the authors names or paper title like here for example: http://colour-science.org/api/0.3.0/html/colour.algebra.interpolation.html#colour.algebra.interpolation.SpragueInterpolator.__call__
- Converting the _._ to _,_.
- Adding the DOI, pages, or url if available/needed.

Do you think we should stick to the original formatting?

**Comment by thomasmansencal** [comment]
Allright, I have started doing the following: I installed Mendeley, pushed my PDF into it and now I'm going through the API to change stuff using its citation format (APA 6th), no emphasis, no nothing:

From:

```
.. [1]  http://www.oscars.org/science-technology/council/projects/aces.html
        (Last accessed 24 February 2014)
.. [2]  `Academy Color Encoding Specification (ACES)
        <https://www.dropbox.com/sh/nt9z9m6utzvkc5m/AACBum5OdkLPCZ3d6trfVeU8a/ACES_v1.0.1.pdf>`_  # noqa
        (Last accessed 24 February 2014)
```

To:

```
References
----------
.. [1]  A.M.P.A.S. (n.d.). Academy Color Encoding System - Science &
        Technology Council | Academy of Motion Picture Arts & Sciences.
        Retrieved February 24, 2014, from
        http://www.oscars.org/science-technology/council/projects/aces.html
.. [2]  A.M.P.A.S, & STC. (2011). Specification S-2008-001 - Academy Color
        Encoding Specification ( ACES ), 1–33. Retrieved from
        https://www.dropbox.com/sh/nt9z9m6utzvkc5m/AACBum5OdkLPCZ3d6trfVeU8a/ACES_v1.0.1.pdf  # noqa
```

**Comment by MichaelMauderer** [comment]
I'm for APA style.
But there are some additional rules about the use for italics for book titles, etc. 
https://www.library.cornell.edu/research/citation/apa

Should we incorporate the appropriate mark-up for that, too?

**Comment by thomasmansencal** [comment]
Yeah definitely!

Found this link that can be useful too: http://libguides.radford.edu/content.php?pid=219451&sid=1822450

**Comment by thomasmansencal** [comment]
Allright! We should have a brand new bibliography! I haven't italicised things in the code although I have done it in the IPython notebooks, we can do a pass for italics once we have merged all the branches together.
